user_management/src/user_management/JWTManager.py

```diff
@@ -8,6 +8,12 @@
 
 
 def user_exist(user_id):
+    if (user_id is None
+            or user_id == ''
+            or type(user_id) != int
+            or user_id < 0
+            or user_id > 2147483647):
```

Comment:
The max value of an id (BigAutoField) is 9223372036854775807. See link below:
https://www.geeksforgeeks.org/bigautofield-django-models/

Reply:
I shall use a macro for this

Reply:
@EthanDelage I'll do the changes because this is relevant, but we might have some other kind of trouble on our minds when we reach 2+ billions users

Reply:
Why don't remove max limit ? If the value is too high, the database won't find it? In Python, there's no max int